guides/v2.1/frontend-dev-guide/themes/debug-theme.md

```diff
@@ -25,6 +25,23 @@ To enable template hints:
 
 ![Enabling template hints]({{ site.baseurl }}/common/images/fdg_debug_theme.png)
 
+{:.bs-callout .bs-callout-info}
+You can also use CLI to enable/disable template hints.
+
+Use `bin/magento dev:template-hints:enable` command in CLI to enable template hints, and `bin/magento dev:template-hints:disable` to disable them. Do not forget to clear cache after command run. For example: 
+
+```bash
+bin/magento dev:template-hints:enable
```

Comment:
Hi @atwixfirster 
This command does not work with Magento v2.1. so can you please update this change for doc v2.2 and v2.3.

Reply:
> This command does not work with Magento v2.1. 

yay!!!

> so can you please update this change for doc v2.2 and v2.3.

**done**

Please review. 

Thank you. @ronak2ram

Reply:
thanks for the quick update @atwixfirster